lib/GeoIP2/Model/AnonymousIP.pm

```diff
@@ -62,11 +62,15 @@ Returns true if the IP address belongs to any sort of anonymous network.
 
 =head2 $anon->is_anonymous_vpn()
 
-Returns true if the IP address belongs to an anonymous VPN system.
+Returns true if the IP address is registered to an anonymous VPN provider.
+If a VPN provider does not register subnets under names associated with them,
+we will likely only flag their IP ranges using the is_hosting_provider
```

Comment:
Since `is_hosting_provider` is a sub name here written inside POD (plain old documentation), I'd prefer to see it wrapped as a code (similar to backticks Markdown), as `C<is_hosting_provider>` . For details, see http://perldoc.perl.org/perlpod.html#Formatting-Codes

Reply:
Thanks for catching this! 603ecf3